beacon_node/beacon_chain/src/light_client_server_cache.rs

```diff
@@ -250,6 +260,9 @@ impl<T: BeaconChainTypes> LightClientServerCache<T> {
             )?;
         }
 
+        *self.latest_current_sync_committee.write() =
```

Comment:
The goal of this variable is to prevent repeated writes, so it should be named `latest_written_current_sync_committee` or `latest_stored_current_sync_committee`, and move the new code inside the `if finalized_period >= sync_committee_period - 1` condition. Otherwise, you may write into `latest_current_sync_committee` indicating it got persisted when not.

Reply:
this should now be resolved: 
8c1f980